theia-swift-docker/Dockerfile

```diff
@@ -1,36 +1,15 @@
 ARG NODE_VERSION=10
 FROM node:$NODE_VERSION as extension-builder
-
 RUN npm install -g vsce
-
 RUN git clone --depth 1 https://github.com/apple/sourcekit-lsp
-
 WORKDIR /sourcekit-lsp/Editors/vscode
 RUN npm install
 RUN npm run postinstall
 RUN vsce package -o ./sourcekit-lsp.vsix
 
 
-FROM satishbabariya/swift as lsp-builder
```

Comment:
I am guessing that the equivalent of this removed block is covered by the satishbabariya/sourcekit-lsp image, used below?

Reply:
yeah, but the satishbabariya/swift image is a nightly snapshot of the swift development branch, and satishbabariya/sourcekit-lsp is based on newly released swift 5.0.1

Reply:
in future, extension-builder will also go away after vscode extension is released